truckersmp_cli/steamruntime_helper.py

```diff
@@ -38,26 +68,23 @@ def main():
     args = arg_parser.parse_args()
 
     if args.verbose is not None and args.verbose > 1:
+        print("Executables (early):", args.early_executable)
         print("Executables:", args.executable)
         print("Game Arguments:", args.game_arguments)
+        print("Waiting time (early):", args.early_wait_before_start)
         print("Waiting time:", args.wait_before_start)
 
-    env = os.environ.copy()
+    early_thirdparty_processes = [] if args.early_executable is None \
+        else start_thirdparty_programs(args.early_executable, args.game_arguments[0:3])
 
-    thirdparty_processes = []
-    if args.executable is not None:
-        env_3rdparty = env.copy()
-        if "LD_PRELOAD" in env_3rdparty:
-            del env_3rdparty["LD_PRELOAD"]
-        for path in args.executable:
-            thirdparty_processes.append(
-                subproc.Popen(
-                    # ["python3", "/path/to/proton", "run"] + ["/path/to/program.exe"]
-                    args.game_arguments[0:3] + [path, ],
-                    env=env_3rdparty, stderr=subproc.STDOUT))
+    time.sleep(args.early_wait_before_start)
+
+    thirdparty_processes = [] if args.executable is None \
+        else start_thirdparty_programs(args.executable, args.game_arguments[0:3])
```

Comment:
This if-else isn't necessary because we get an empty array by default as a return value if `args.executable` is an empty array.

Reply:
Small details, imho now it's more clean.